core/src/Cabana_DeepCopy.hpp

```diff
@@ -126,6 +126,66 @@ inline void deep_copy(
     }
 }
 
+//---------------------------------------------------------------------------//
+namespace Experimental
+{
+//---------------------------------------------------------------------------//
+/*!
+  \brief Create a mirror view of the given AoSoA in the given memory
+  space. Same space specialization returns the input AoSoA.
+
+  \note The semantics of using the word view in the name of this function
+  indicate that memory allocation will only occur if the requested mirror
```

Comment:
\note ~~The semantics of using the word view in the name of this function
	  indicate that~~ (M)emory allocation will only occur if the requested mirror

Reply:
done